hikari/internal/enums.py

```diff
@@ -140,14 +140,20 @@ def __new__(
             "_member_names_": (member_names := []),
             # Required to be immutable by enum API itself.
             "__members__": types.MappingProxyType(namespace.names_to_values),
-            **namespace,
             **{
                 name: value
                 for name, value in Enum.__dict__.items()
                 if name not in ("__class__", "__module__", "__doc__")
             },
         }
 
+        # We don't want to override the __str__ behaviour inherited from builtins.str for string based enums.
+        if issubclass(base, str):
+            new_namespace.pop("__str__", None)
+
```

Comment:
Why not? What difference does it make?

Reply:
This is the difference between a string enum value acting as its value rather than its name